modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/search/test/DLFileEntryIndexerLocalizedContentTest.java

```diff
@@ -92,18 +91,9 @@ public void testJapaneseContent() throws Exception {
 		List<String> contentStrings = new ArrayList<>(
 			Collections.singletonList("content_ja_JP"));
 
-		String word1 = "新規";
-		String word2 = "作成";
+		assertLocalization(contentStrings, _search("新規", LocaleUtil.JAPAN));
 
-		Stream.of(
-			word1, word2
-		).forEach(
-			searchTerm -> {
-				Document document = _search(searchTerm, LocaleUtil.JAPAN);
-
-				assertLocalization(contentStrings, document);
-			}
-		);
+		assertLocalization(contentStrings, _search("作成", LocaleUtil.JAPAN));
```

Comment:
@slnn Can you check if SF complains when you remove the empty above this line? If not, please remove the empty line here and below.